test/mediastream.js

```diff
@@ -159,6 +159,25 @@ tape('.removeTrack and .addTrack on remote MediaStream', function(t) {
   });
 });
 
+tape('.enabled on MediaStreamTrack', function(t) {
+  return getMediaStream().then(function(stream) {
+    var tracks = stream.getTracks();
+    tracks.forEach(function(track) {
+      track.enabled = false;
+    });
+    t.ok(tracks.every(function(track) {
+      return !track.enabled;
+    }), 'enabled is false on all MediaStreamTracks');
+    tracks.forEach(function(track) {
+      track.enabled = true;
+    });
+    t.ok(tracks.every(function(track) {
+      return track.enabled;
+    }), 'all MediaStreamTracks are enabled again');
+    t.end();
```

Comment:
@taylorhoward92 I forgot something in my initial review: we should stop each MediaStreamTrack; otherwise, the Node process may hang.